### PR TITLE
Update let-users-reset-passwords.md

### PR DESCRIPTION
**Pull request by tilarso**
Added missed step of selecting number of methods + which methods for proper SSPR configuration.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/admin/add-users/let-users-reset-passwords.md [hunk 1]
```diff
@@ -46,9 +46,10 @@ Check out this video and others on our [YouTube channel](https://go.microsoft.co
 > [!VIDEO https://www.microsoft.com/videoplayer/embed/RE3AY8S]
 
 1. In the Microsoft 365 admin center, in the left navigation pane, select **Settings** > **Org settings**, and then <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">**Security & privacy**</a>.
-1. Under **Let your people reset their own passwords**, select**Azure AD admin center**.
+1. Under **Self-service password reset**, select **Go to the Azure portal to turn on self-service password reset**.
 1. In the left navigation pane, select **Users**, and then on the **Users - all users** page, select **Password reset**.
 1. Select **All** to enable self-service password reset, and then select **Save**.
+1. In the left navigatino pane, select **Authentication methods**, and select the **Number of methods required to reset**, and desired **Methods available to users**, and then click **Save**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/let-users-reset-passwords.md]:
```suggestion
1. In the left navigation pane, select **Authentication methods** and select the **Number of methods required to reset** and desired **Methods available to users**, and then select **Save**. 
```

File: microsoft-365/admin/add-users/let-users-reset-passwords.md [hunk 2]
```diff
@@ -77,8 +78,10 @@ These steps turn on self-service password reset for everyone in your business.
 4. Under **Self-service password reset**, select **Go to the Azure portal to turn on self-service password reset**.
 
 5. On the **Properties** page, select **All** to enable it for everyone in your business, and then select **Save**.
+
+6. In the left navigatino pane, select **Authentication methods**, and select the **Number of methods required to reset**, and desired **Methods available to users**, and then click **Save**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/let-users-reset-passwords.md]:
```suggestion
6. In the left navigation pane, select **Authentication methods** and select the **Number of methods required to reset** and desired **Methods available to users**, and then select **Save**. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @tilarso, kindly commit the suggested changes. Many thanks.

**Comment by tilarso** [comment]
@JohanFreelancer9 - appreciate the review. All suggestions committed and ready for merge to master.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**